pkgs/applications/misc/far2l/default.nix

```
@@ -1,29 +1,22 @@
 { stdenv, fetchFromGitHub, fetchpatch, makeWrapper, cmake, pkgconfig, wxGTK30, glib, pcre, m4, bash,
-  xdg_utils, gvfs, zip, unzip, gzip, bzip2, gnutar, p7zip, xz, imagemagick, darwin }:
+  xdg_utils, gvfs, zip, unzip, gzip, bzip2, gnutar, p7zip, xz, imagemagick, darwin, libuchardet, spdlog, xercesc, fmt, openssl, libssh, samba, neon, libnfs, libarchive }:
```

```suggestion
, xdg_utils, gvfs, zip, unzip, gzip, bzip2, gnutar, p7zip, xz, imagemagick, darwin, libuchardet
, spdlog, xercesc, fmt, openssl, libssh, samba, neon, libnfs, libarchive }:
```

Also please do not unput darwin directly.

---

`darwin` is in prior code written by package maintainer. I won't dare change this. Especially that it's for Mac I which I can't test.

---

Wrapping I will do, just want to see the oborg checks complete before resetting by push.